### PR TITLE
Yearly email to backers

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

**Comment by piamancini** [comment]
looks great. I love the tweet with the gift, great idea!
The first phrase reads a bit clunky to me, how about:
- As 2016 reaches it's end <emoji>, let's be frank, we can't wait for it to be over <emoji>
- 2016 is ending <emoji>. Let's be frank...
- 2016 is finally ending <emoji> and let's be frank...

**Comment by asood123** [comment]
Nicely phrased, inspiring, feel good email. Dig the tweet. Agree that first
sentence sounds off.

**Comment by xdamman** [comment]
What about: "2016 is finally ending 🎉. And frankly, can't wait for this year to be over 😱 ."

@asood123 can you review the PR ?

File: cron/yearly/email-report.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  } else if (totalCollectives === 1) {
+    tweet = `🎁 In 2016, I have contributed ${collectivesDonationsNames[0].replace(' to ',' to the ')} collective`;
+  } else {
+    tweet = `🎁 In 2016, I have contributed ${formatArrayToString(collectivesDonationsNames)}'s collective`;
+  }
+
+  return tweet;
+}
+
+const processUser = (user) => {
```
Comment (asood123) [inline cron/yearly/email-report.js]: It's hard to follow this function. Can you leave comments with the logic in the future?

File: templates/emails/group.yearlyreport.hbs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+</center>
+</div>
+{{/each}}
+</div>
+
+<p>
+<b>📍Don't forget:</b><br />
+- You can discover other great collectives to support on <a href="https://opencollective.com/discover?{{utm}}">https://opencollective.com/discover</a><br />
+- You can manage your subscriptions on <a href="https://opencollective.com/subscriptions?{{utm}}">https://opencollective.com/subscriptions</a><br />
+- We are an open collective ourselves and <a href="https://github.com/opencollective/opencollective">our code is available on Github</a> under MIT license. We welcome contributions of any kind! 😊
```
Comment (asood123) [inline templates/emails/group.yearlyreport.hbs]: fyi, GitHub has a capital H. It's confusing, thus avoiding "OpenCollective". :)

Comment (xdamman) [inline templates/emails/group.yearlyreport.hbs]: So? You want me to put "GitHub" instead of "Github"?
